models/sessions/scratch/snowplow_mobile_sessions_this_run.sql

```diff
@@ -28,9 +28,9 @@ select
 
   {% if var('snowplow__session_stitching') %}
     -- updated with mapping as part of post hook on derived sessions table
-    cast(es.device_user_id as {{dbt_utils.type_string() }}) as stitched_user_id,
+    cast(es.device_user_id as {{snowplow_utils.type_string(4096) }}) as stitched_user_id,
```

Comment:
Can we apply the same macro to the `snowplow_mobile_incremental_manifest` as you have done in the web release?